### ISSUE TITLE
Runtime crypter

### ISSUE THREAD

**Issue by IgorAlaf** [UnamSanctam/SilentCryptoMiner#2285]
Is runtime crypter suitable for encrypting a miner?Thanks

**Comment by UnamSanctam** [comment]
Yes, though the crypter would need to make the file undetected as well.

**Comment by UnamSanctam** [comment]
The detections are from the crypter, so using another crypter (or using multiple if the crypter supports it, since it sounds like yours is specifically for runtime detections) would be the solution.
